print_docs.py

```diff
@@ -268,6 +268,18 @@ def html_head(title):
         <link rel="shortcut icon" href="https://leanprover-community.github.io/assets/img/lean.ico">
         <title>mathlib docs: {1}</title>
         <meta charset="UTF-8">
+        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
```

Comment:
What is this `polyfill` thing? Is there a reason to grab mathjax from there? Should we self-host a copy?

Reply:
The MathJax docs said to put both these inclusions which surprised me. Polyfill is apparently there as a backoff for "older browsers". I don't know how necessary it really is, I'll find out. As for self-hosting I don't know all the tradeoffs. The authors of mathlib don't need to care of course.

Reply:
The docs http://docs.mathjax.org/en/latest/output/browser.html say that polypill is there to support IE11. The stats here https://www.w3counter.com/globalstats.php say that IE11 is ranked 8th with 2.3% of... whatever they're counting.

Reply:
Ok, thanks for explaining. In that case, I'm fine with using polyfill.

Reply:
Thanks for the question. I only followed up with that research once you asked, having merrily copied and pasted it.